### PR TITLE
Add sleep/wake face from Mark II Skill

### PR DESCRIPTION
**Pull request by krisgesling**
#### Description
This moves the sleep/wake face from the Mark II Skill into the
naptime Skill where the actions are performed.

It also includes a refactor to tidy up and segregate actions based
on the platform they are running on.

Will require companion PR in Mark II Skill - do not merge in isolation.

#### Type of PR
- [ ] Bugfix
- [x] Feature implementation
- [x] Refactor of code (without functional changes)
- [ ] Documentation improvements
- [ ] Test improvements

#### Testing
Hey Mycroft, go to sleep
Hey Mycroft, wake up

#### CLA
- [x] Yes

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -47,44 +53,50 @@ def handle_go_to_sleep(self, message):
...
+            time.sleep(0.15)
+        self.enclosure.eyes_look("d")
 
-    def wake_up_animation(self):
+    def display_waking_face(self) -> None:
+        """Display the waking face depending on the platform."""
+        if self.gui.connected:
+            self.gui.remove_page("resting.qml")
+            self.gui.show_page("awake.qml", override_idle=5)
+            # TODO this shouldn't be necessary - remove 2 lines when fixed.
```
Comment (chrisveilleux) [inline __init__.py]: could you add more information here?  what is the issue?  what is the fix?

File: ui/Face.qml [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import QtQuick.Layouts 1.4
```
Comment (chrisveilleux) [inline ui/Face.qml:1]: should there be a Mark II version of this file that uses the new design best practices?

**Review by chrisveilleux — APPROVED** [review]
two small comments.

File: ui/Face.qml (hunk 2 above)
Comment (krisgesling) [reply ui/Face.qml:1]: Maybe - though the current version looks good to me, and I don't believe there's a Mark II grid design in Figma. So I'm suggesting that's out of scope for this PR.

File: __init__.py (hunk 1 above)
Comment (krisgesling) [reply __init__.py]: Improved the comment, but also a good reminder.

@AIIX any idea why an `override_idle=5` wouldn't stop showing the page after 5 seconds?

Comment (AIIX) [reply __init__.py]: I know what causes it, I'm not sure why that is implemented this way tho, @forslund can maybe tell, Comment out the condition on L427 to L429 till the try exception in mark-2 skill https://github.com/MycroftAI/skill-mark-2/blob/a39ce4d8f6b4469bf6e61e55ae314bf42bf1bbfb/__init__.py#L427

Comment (krisgesling) [reply __init__.py]: Thanks, I've flagged it in that repo and added it to the Skills Interaction sprint.

**Pull request merged**